modules/tinymce/src/core/main/ts/fmt/RemoveFormat.ts

```diff
@@ -471,9 +471,14 @@ const remove = (ed: Editor, name: string, vars?: FormatVars, node?: Node | Range
   const isRemoveBookmarkNode = (node: Node) => Bookmarks.isBookmarkNode(node) && NodeType.isElement(node) && (node.id === '_start' || node.id === '_end');
 
   // Merges the styles for each node
-  const process = (node: Node) => {
+  const process = (nodes: Node[], node: Node) => {
     let lastContentEditable: boolean, hasContentEditableState: boolean;
 
+    // TINY-6567 Include the last node in the selection
+    if (NodeType.isText(node) && FormatUtils.hasBlockChildren(dom, node.parentNode)) {
+      removeFormat(ed, format, vars, node.parentNode, node.parentNode);
```

Comment:
```suggestion
    const parentNode = node.parentNode;
    if (NodeType.isText(node) && FormatUtils.hasBlockChildren(dom, parentNode)) {
      removeFormat(ed, format, vars, parentNode, parentNode);
```
I missed this last time sorry, but doing this will allow better minification and removes a little duplication.

Reply:
updated